protocol/query/method/message/data/dataElect.json

```
@@ -13,7 +13,7 @@
         "instance_id": {
             "type": "string",
             "contentEncoding": "base64",
-            "$comment": "Hash : SHA256('consensus'||created_at||key:type||key:id||key:property||value)"
+            "$comment": "Hash : SHA256('consensus'||key:type||key:id||key:property)"
```

The protocol documentation defines the hashing function as [`HashLen` ](https://github.com/dedis/student_21_pop/blob/master/docs/protocol.md#concatenation-for-hashing).

```suggestion
            "$comment": "Hash : HashLen('consensus', key:type, key:id, key:property)"
```